File: 2022-2023/sourcecode/lab/lab6/TaskyJava/app/src/main/java/pl/udu/uwr/pum/taskyjava/data/DataProvider.java
package pl.udu.uwr.pum.taskyjava.data;

import java.util.ArrayList;

import pl.udu.uwr.pum.taskyjava.model.Task;
import pl.udu.uwr.pum.taskyjava.model.TaskGroup;

final public class DataProvider {
    private DataProvider(){}

    public static ArrayList<Task> getTasks(){
        ArrayList<Task> tasks = new ArrayList<>();
            tasks.add(new Task("task1", new TaskGroup("Group 1")));
            tasks.add(new Task("task2", new TaskGroup("Group 1")));
            tasks.add(new Task("task3", new TaskGroup("Group 1")));
            tasks.add(new Task("task4", new TaskGroup("Group 1")));
            tasks.add(new Task("task5", new TaskGroup("Group 1")));
            tasks.add(new Task("task a", new TaskGroup("Group 2")));
            tasks.add(new Task("task b", new TaskGroup("Group 2")));
            tasks.add(new Task("task c", new TaskGroup("Group 2")));
            tasks.add(new Task("task d", new TaskGroup("Group 2")));
            return tasks;
    }
}

File: 2022-2023/sourcecode/lab/lab6/TaskyKotlin/app/src/main/java/pl/udu/uwr/pum/taskykotlin/data/DataProvider.kt
package pl.udu.uwr.pum.taskykotlin.data

import pl.udu.uwr.pum.taskykotlin.model.Task
import pl.udu.uwr.pum.taskykotlin.model.TaskGroup
import pl.udu.uwr.pum.taskykotlin.model.TaskRow

object DataProvider {
    val dummyData = listOf(
        Task("task1", TaskGroup("Group 1")),
        Task("task2", TaskGroup("Group 1")),
        Task("task3", TaskGroup("Group 1")),
        Task("task4", TaskGroup("Group 1")),
        Task("task5", TaskGroup("Group 1")),
        Task("task a", TaskGroup("Group 2")),
        Task("task b", TaskGroup("Group 2")),
        Task("task c", TaskGroup("Group 2")),
        Task("task d", TaskGroup("Group 2"))
    )
}